.flake8

```diff
@@ -2,6 +2,7 @@
 max-line-length = 120
 max-complexity = 10
 ignore =
+    E402,  # module level import not at top of file
```

Comment:
Is it able to disable at the line-level to minimize the impact?

Reply:
updated